enterprise/backend/src/metabase_enterprise/metabot_v3/dummy_tools.clj

```diff
@@ -0,0 +1,75 @@
+(ns metabase-enterprise.metabot-v3.dummy-tools
+  (:require
+   [cheshire.core :as json]
+   [metabase-enterprise.metabot-v3.tools.create-dashboard-subscription]
+   [metabase-enterprise.metabot-v3.tools.query]
+   [metabase-enterprise.metabot-v3.tools.who-is-your-favorite]
+   [metabase.api.common :as api]
+   [toucan2.core :as t2]))
+
+(defn- get-current-user
+  [_ _ context]
+  {:output (if-let [{:keys [id email first_name last_name]}
+                    (or (some-> api/*current-user* deref)
+                        (t2/select-one [:model/User :id :email :first_name :last_name] api/*current-user-id*))]
+             {:id id
+              :name (str first_name " " last_name)
+              :email_address email}
+             {:error "current user not found"})
+   :context context})
+
+(defn- get-dashboard-details
+  [_ {:keys [dashboard_id]} context]
+  {:output (or (t2/select-one [:model/Dashboard :id :description :name] dashboard_id)
+               {:error "dashboard not found"})
+   :context context})
+
+(defn- dummy-tool-messages
+  [tool-id arguments content]
+  (let [call-id (random-uuid)]
+    [{:content    nil
+      :role       :assistant,
+      :tool_calls [{:id        call-id
+                    :name      tool-id
+                    :arguments (json/generate-string arguments)}]}
+
+     {:content      (json/generate-string content)
+      :role         :tool,
+      :tool_call_id call-id}]))
+
+(defn- dummy-get-current-user
+  [context]
+  (let [content (:output (get-current-user :get-current-user {} context))]
+    (dummy-tool-messages :get-current-user {} content)))
+
+(defn- dummy-get-dashboard-details
+  [context]
+  (reduce (fn [messages viewed]
+            (if-let [dashboard-id (when (= (:type viewed) :dashboard)
```

Comment:
I'm assuming we'd have an `if-let` for each dummy tool we wrote (eg, query, x-ray, permissions group, etc)?